frontend/server/src/DAO/RunsGroups.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace OmegaUp\DAO;
+
+/**
+ * RunsGroups Data Access Object (DAO).
+ *
+ * Esta clase contiene toda la manipulacion de bases de datos que se necesita
+ * para almacenar de forma permanente y recuperar instancias de objetos
+ * {@link \OmegaUp\DAO\VO\RunsGroups}.
+ *
+ * @access public
+ */
+class RunsGroups extends \OmegaUp\DAO\Base\RunsGroups {
+    /**
+     * @return list<array{contest_score: float|null, identity_id: int, penalty: int, problem_id: int, score: float, type: null|string}>
+     */
+    final public static function getProblemsetRunsGroups(
+        int $problemsetId,
+        string $penaltyType = 'MAX'
+    ): array {
+        $penaltyQuery = 'MAX(r.penalty) AS penalty';
+        if ($penaltyType != 'MAX') {
+            $penaltyQuery = 'SUM(r.penalty) AS penalty';
+        }
+        $sql = "SELECT
+                    IFNULL(SUM(mspg.score), 0.0) AS score,
+                    SUM(mspg.score) * pp.points AS contest_score,
```

Comment:
```suggestion
                    IFNULL(SUM(mspg.score) * pp.points, 0) AS contest_score,
```

así podemos evitar los `NULL`s en esta columna.

Reply:
Listo